packages/editor/src/plugins/fill-in-the-blanks-exercise/blank-renderer-static.tsx

```diff
@@ -0,0 +1,66 @@
+import { ChangeEventHandler, useContext } from 'react'
+
+import type { FillInTheBlanksMode } from '.'
+import { DraggableSolution } from './components/blank-solution'
+import { DroppableBlank } from './components/droppable-blank'
+import { FillInTheBlanksContext } from './context/blank-context'
+import { setTextUserTypedIntoBlank } from './util/handlers'
+import { cn } from '@/helper/cn'
+
+interface BlankRendererStaticProps {
+  blankId: string
+  forceMode?: FillInTheBlanksMode
+  onChange?: ChangeEventHandler<HTMLInputElement>
+}
+
+/**
+ * Renders either an input element (where user can type into)
+ * or a drop area (where user can drop draggable answers),
+ * depending on the mode
+ */
+export function BlankRendererStatic(props: BlankRendererStaticProps) {
+  const { blankId, forceMode, onChange } = props
+
+  const context = useContext(FillInTheBlanksContext)
+  if (context === null) return null
+
+  const mode = forceMode ?? context.mode
+  const draggableSolution = [...context.locationOfDraggables.value].find(
+    (entry) => entry[1] === blankId
+  )
+  const draggableId = draggableSolution?.[0] ?? null
+  const draggable = context.draggables.find(
+    (draggable) => draggable.draggableId === draggableId
+  )
+  const draggableText = draggable?.text ?? ''
+
+  const feedback = context.feedbackForBlanks
+  const isAnswerCorrect = feedback.get(blankId)?.isCorrect
+  const text = context.textInBlanks.get(blankId)?.text ?? ''
+
+  return mode === 'typing' ? (
```

Comment:
(out of scope here, but arrow keys to move to the next blank might be a nice addition for users as well)

Reply:
Could you maybe open an issue, explaining how you imagine that working? I guess we wouldn't just be using simple arrow left/right?